src/components/Calendario.elements.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
+import { colors } from '../theme';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
import { colors } from "../theme";
```